vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/overview/components/ProjectCollection.java

```diff
@@ -0,0 +1,191 @@
+  private Div titleSection = new Div();
+  @Serial
+  private static final long serialVersionUID = 8579375312838977742L;
+  final TextField projectSearchField = new TextField();
+  final Grid<ProjectPreview> projectGrid = new Grid<>(ProjectPreview.class, false);
+  final Button createProjectButton = new Button("Add");
+  private final String title;
+  private final ZoneId clientZoneId;
+  private final ProjectInformationService projectInformationService;
+  private final List<ComponentEventListener<ProjectCreationClickedEvent>> projectCreationClickedListeners = new ArrayList<>();
+
+  private ProjectCollection(String title, ZoneId clientZoneId,
+      ProjectInformationService projectInformationService) {
+    this.title = title;
+    this.clientZoneId = clientZoneId;
+    this.projectInformationService = projectInformationService;
+    layoutComponent();
+    createLazyProjectView();
+    configureSearch();
+    configureProjectCreationButton();
+  }
+
+  private void layoutComponent() {
+    addClassNames("projects-collection");
+    layoutSearchField();
+    layoutProjectCreationButton();
+    layoutGrid();
+    layoutTitleSection();
+    layoutControlSection();
+    layoutGridSection();
+    controlSection.add(projectSearchField);
+    controlSection.add(createProjectButton);
+
+    gridSection.add(projectGrid);
+    add(titleSection);
+    add(controlSection);
+    add(gridSection);
+  }
+
+  private void createLazyProjectView() {
+    projectPreviewGridLazyDataView = projectGrid.setItems(query -> {
+      List<SortOrder> sortOrders = query.getSortOrders().stream().map(
+              it -> new SortOrder(it.getSorted(), it.getDirection().equals(SortDirection.DESCENDING)))
+          .collect(Collectors.toList());
+      // if no order is provided by the grid order by last modified (the least priority)
+      sortOrders.add(SortOrder.of("lastModified").descending());
+      return projectInformationService.queryPreview(projectPreviewFilter, query.getOffset(),
+          query.getLimit(), List.copyOf(sortOrders)).stream();
+    });
+  }
+
+  private void configureSearch() {
+    projectSearchField.setValueChangeMode(ValueChangeMode.LAZY);
+    projectSearchField.addValueChangeListener(event -> {
+      projectPreviewFilter = event.getValue().trim();
+      projectPreviewGridLazyDataView.refreshAll();
+    });
+  }
+
+  private void configureProjectCreationButton() {
+    createProjectButton.addClickListener(listener -> {
+      fireClickEvent();
+    });
+  }
+
+  private void layoutSearchField() {
+    projectSearchField.setPlaceholder("Search");
+    projectSearchField.setClearButtonVisible(true);
+    projectSearchField.setSuffixComponent(VaadinIcon.SEARCH.create());
+    projectSearchField.addClassNames("search-field");
+  }
+
+  private void layoutProjectCreationButton() {
+    createProjectButton.addClassName("add-project-button");
+  }
+
+  private void layoutGrid() {
```

Comment:
Does not only layout the Grid but also determines how it is filled. 
suggesiton: rename to buildGrid